cmd/crictl/util.go

```diff
@@ -289,37 +289,54 @@ func outputStatusInfo(status, handlers string, info map[string]string, format st
 	}
 	sort.Strings(keys)
 
-	jsonInfo := "{" + "\"status\":" + status + ","
+	infoMap := map[string]interface{}{}
```

Comment:
Just a question about style/preference: should we already move to use **any** over **interface{}** here?

Reply:
I would say no, because there is a somewhat expected format for the status. I'm a bit on the fence about this topic, though.

Reply:
So, you prefer to keep **interface{}**?

Just checking as any is a type alias for an empty interface, so both things are the same.

Reply:
Ah, yeah I can change that!